src/shelly_cct_controller.hpp

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) Shelly-HomeKit Contributors
+ * All rights reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "mgos.hpp"
+#include "mgos_hap_chars.hpp"
+#include "mgos_timers.hpp"
+#include "shelly_light_bulb_controller.hpp"
+#include "shelly_output.hpp"
+
+#pragma once
+
+namespace shelly {
+
+struct StateCCT {
+  float ww = 0;
+  float cw = 0;
+
+  StateCCT operator+(const StateCCT &other) {
+    StateCCT s;
+    s.ww = ww + other.ww;
+    s.cw = cw + other.cw;
+    return s;
```

Comment:
try this: `return {.ww = ww + other.ww, .cw = cw + other.cw};`, here and below.

Reply:
done;
but apparently you can only chose one: Aggregate initialization or default member initializers... so I had to get rid of the initializers for now